internal/common/function/function.go

```diff
@@ -171,6 +171,9 @@ func ValidateRestoreSpec(ctx context.Context, clientInstance client.Client, nonA
 func ValidateBslSpec(ctx context.Context, clientInstance client.Client, nonAdminBsl *nacv1alpha1.NonAdminBackupStorageLocation) error {
 	// TODO Introduce validation for NaBSL as described in the
 	// https://github.com/migtools/oadp-non-admin/issues/146
+	if nonAdminBsl.Spec.BackupStorageLocationSpec.Default {
+		return fmt.Errorf("NonAdminBackupStorageLocation can not set spec.bslSpec.default to true")
```

Comment:
its not `bslSpec` https://github.com/migtools/oadp-non-admin/blob/b5126fddf738c5f3155b902b9a4981a47aee15dc/api/v1alpha1/nonadminbackupstoragelocation_types.go#L36

and would rewrite message to something like this
```
NonAdminBackupStorageLocation spec.backupStorageLocationSpec.default can not be set to true
```

Reply:
+1

Reply:
changed to:
`NonAdminBackupStorageLocation cannot be used as a default BSL`